Documentation/helm-operator.md

```diff
@@ -148,6 +148,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `admissionController.tolerations`   | Array of tolerations in YAML format which will be added to admission controller deployment.                                 | <none>                                                    |
 | `admissionController.nodeAffinity`  | The node labels for affinity of the admission controller deployment (***)                                                   | <none>                                                    |
 | `allowMultipleFilesystems`          | **(experimental in  Octopus (v15))** Allows multiple filesystems to be deployed to a Ceph cluster.                          | `false`                                                   |
+| `monitoring.enabled`                | Create necessary RBAC rules for Rook to integrate with prometheus monitoring. Requires Prometheus to be pre-installed. | `false` |
```

Comment:
Maybe rename this to `monitoring.rbacEnable`? It's only about RBAC currently.

Reply:
I would tend to agree, but this is the same naming used for the `rook-ceph-cluster` chart. I think keeping consitency might be preferred over a better name.

Reply:
It creates the same exact resources as the cluster chart for monitoring.enabeld? If so, makes sense to me that it would be named the same